### PR TITLE
Home Styling

### PR DESCRIPTION
**Pull request by CesarAlarcon-1983**

### CODE REVIEW

File: src/_modules/donate-cta/donate-cta.jade [hunk 1]
```diff
@@ -1,12 +1,14 @@
 mixin donate-cta(spec)
-    section.donate-cta
+    .donate-cta
```
Comment (migueljrojas) [inline src/_modules/donate-cta/donate-cta.jade]: Add section tag to the mixin base class

File: src/_modules/donate-cta/donate-cta.scss [hunk 2]
```diff
@@ -1 +1,40 @@
-// donate-cta styles
+.donate-cta{
+  @include padd();
+  background: $lightGray;
+  display: block;
```
Comment (migueljrojas) [inline src/_modules/donate-cta/donate-cta.scss]: not need of display block

File: src/_modules/donate-cta/donate-cta.scss [hunk 3]
```diff
@@ -1 +1,40 @@
...
+  }
+
+  &__contact-wrapper {
+    display: inline-block;
+    text-align: center;
+
+    @include mq(md) {
+      padding-left: 50px;
+    }
+
```
Comment (migueljrojas) [inline src/_modules/donate-cta/donate-cta.scss]: remove whitespace

File: src/_modules/footer/footer.jade [hunk 4]
```diff
@@ -1,13 +1,16 @@
...
-                a(href="+54 11 4708-4108") 4708-4108
-                span Vos tambien podes sumarte!
-                .footer__container__contact__buttons
-                    a VOLUNTARIADO
-                    a DONACIONES
-                            
-            .footer__container__copyright
-                span La merced vida &copy; 2017, Todos los derechos reservados
+                    
+        .footer__contact__container
```
Comment (migueljrojas) [inline src/_modules/footer/footer.jade]: - Remove container suffix
- Check irregular indentation

File: src/_modules/footer/footer.jade [hunk 5]
```diff
@@ -1,13 +1,16 @@
...
+          
+          .footer__contact__container__data
+            i.fa.fa-phone
+            a.footer__contact__container__data__phone(href="+54 11 4708-4108") 4708-4108
+            span.footer__contact__container__data__text Vos tambien podes sumarte!
+            
+          .footer__contact__buttons
+            .footer__contact__buttons__container
```
Comment (migueljrojas) [inline src/_modules/footer/footer.jade]: - .footer__contact__buttons__container is unnecessary, assign class to anchors

File: src/_modules/footer/footer.scss [hunk 6]
```diff
@@ -1,3 +1,112 @@
 .footer {
-    
+  background: transparentize($mainColor, 0.9);
+
+  &__contact {
+    justify-content: center;
```
Comment (migueljrojas) [inline src/_modules/footer/footer.scss:6]: unneeded as there's no display flex

File: src/_modules/footer/footer.scss [hunk 7]
```diff
@@ -1,3 +1,112 @@
...
+      max-width: 900px;
+      margin-left: auto;
+      margin-right: auto;
+      justify-content: center;
+      padding: 8px;
+      margin-bottom: -20px;
+
+      @include mq(md) {
+        flex-wrap: nowrap;
+      transform: translateY(-50%);
```
Comment (migueljrojas) [inline src/_modules/footer/footer.scss]: check indent

File: src/_modules/footer/footer.scss [hunk 8]
```diff
@@ -1,3 +1,112 @@
...
+          order: 2;
+          color: $white;
+          font-size: 24px;
+          align-self: center;
+          padding: 0 10px;
+          @include mq(md) {
+            order: 1;
+            padding: 0 10px;
+            font-size: 36px;
+
```
Comment (migueljrojas) [inline src/_modules/footer/footer.scss]: check blank line

File: src/_modules/footer/footer.scss [hunk 9]
```diff
@@ -1,3 +1,112 @@
...
+        @include mq(md) {
+          flex-wrap: nowrap;
+        }
+
+        i {
+          order: 2;
+          color: $white;
+          font-size: 24px;
+          align-self: center;
+          padding: 0 10px;
```
Comment (migueljrojas) [inline src/_modules/footer/footer.scss:41]: separate the include from the style rules

File: src/_modules/footer/footer.scss [hunk 10]
```diff
@@ -1,3 +1,112 @@
...
+            display: flex;
+            background: $white;
+            color: $mainColor;
+            justify-content: center;
+          }
+
+          &:first-of-type {
+            margin-bottom: 5px;
+          }
+
```
Comment (migueljrojas) [inline src/_modules/footer/footer.scss:92]: blank line

File: src/_styles/pages/_home.scss [hunk 11]
```diff
@@ -1,10 +1,303 @@
...
-            position: absolute;
-            top: 100px;
+      &__image{
+
+        @include img(120%, 100%, 100%, auto);
+
+        @include mq(md) {
+          @include img(40%);
+        }
+
```
Comment (migueljrojas) [inline src/_styles/pages/_home.scss]: check blank lines

File: src/_styles/pages/_home.scss [hunk 12]
```diff
@@ -1,10 +1,303 @@
...
+      }
+
+      &__data {
+        position: absolute;
+        top: 5%;
+        padding: 20px;
+        width: 100%;
+
+
```
Comment (migueljrojas) [inline src/_styles/pages/_home.scss]: extra blank line

File: src/_styles/pages/_home.scss [hunk 13]
```diff
@@ -1,10 +1,303 @@
...
+        @include mq(md) {
+          max-width: 590px;
+          top: 25%;
+          right: 50%;
+          width: 50%;
+
```
Comment (migueljrojas) [inline src/_styles/pages/_home.scss]: blank line

File: src/_styles/pages/_home.scss [hunk 14]
```diff
@@ -1,10 +1,303 @@
...
+        }
+
+        &__text {
+          text-align: center;
+        }
+      }
+    }
+
+  &__donation {
+
```
Comment (migueljrojas) [inline src/_styles/pages/_home.scss]: remove

**Review by migueljrojas — CHANGES_REQUESTED** [review]

File: src/_modules/donate-cta/donate-cta.jade (hunk 1 above)
Comment (CesarAlarcon-1983) [reply src/_modules/donate-cta/donate-cta.jade]: Done

File: src/_modules/donate-cta/donate-cta.scss (hunk 2 above)
Comment (CesarAlarcon-1983) [reply src/_modules/donate-cta/donate-cta.scss]: Done

File: src/_modules/donate-cta/donate-cta.scss (hunk 3 above)
Comment (CesarAlarcon-1983) [reply src/_modules/donate-cta/donate-cta.scss]: Done

File: src/_modules/footer/footer.jade (hunk 4 above)
Comment (CesarAlarcon-1983) [reply src/_modules/footer/footer.jade]: Done

File: src/_modules/footer/footer.jade (hunk 5 above)
Comment (CesarAlarcon-1983) [reply src/_modules/footer/footer.jade]: Done

File: src/_modules/footer/footer.scss (hunk 6 above)
Comment (CesarAlarcon-1983) [reply src/_modules/footer/footer.scss:6]: Done. Display flex added

File: src/_modules/footer/footer.scss (hunk 7 above)
Comment (CesarAlarcon-1983) [reply src/_modules/footer/footer.scss]: Done

File: src/_modules/footer/footer.scss (hunk 8 above)
Comment (CesarAlarcon-1983) [reply src/_modules/footer/footer.scss]: Done

File: src/_modules/footer/footer.scss (hunk 9 above)
Comment (CesarAlarcon-1983) [reply src/_modules/footer/footer.scss:41]: Done

File: src/_modules/footer/footer.scss (hunk 10 above)
Comment (CesarAlarcon-1983) [reply src/_modules/footer/footer.scss:92]: Done

File: src/_styles/pages/_home.scss (hunk 11 above)
Comment (CesarAlarcon-1983) [reply src/_styles/pages/_home.scss]: Done

File: src/_styles/pages/_home.scss (hunk 13 above)
Comment (CesarAlarcon-1983) [reply src/_styles/pages/_home.scss]: Done

File: src/_styles/pages/_home.scss (hunk 12 above)
Comment (CesarAlarcon-1983) [reply src/_styles/pages/_home.scss]: Done

File: src/_styles/pages/_home.scss (hunk 14 above)
Comment (CesarAlarcon-1983) [reply src/_styles/pages/_home.scss]: Done

**Review by migueljrojas — APPROVED** [review]

**Pull request merged**